mmv1/Gemfile.lock

```diff
@@ -6,6 +6,7 @@ GEM
       i18n (>= 1.6, < 2)
       minitest (>= 5.1)
       tzinfo (~> 2.0)
+      zeitwerk (~> 2.3)
```

Comment:
What is the reason to add `zeitwerk`?

Reply:
It was here, because of my misconfiguration. Removed, thanks for noticing!